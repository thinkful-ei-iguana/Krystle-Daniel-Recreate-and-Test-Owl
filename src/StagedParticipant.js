import React from 'react';

function StagedParticipant(props) {
  return (
    <div className="staged-participant">
      {props.name}
    </div>
  );
}

export default StagedParticipant;