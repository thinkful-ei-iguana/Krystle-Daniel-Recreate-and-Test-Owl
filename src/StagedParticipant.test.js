import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import StagedParticipant from './StagedParticipant';


describe('Staged Participant', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<StagedParticipant name="NAME"/>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<StagedParticipant name="NAME"/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });

 
});