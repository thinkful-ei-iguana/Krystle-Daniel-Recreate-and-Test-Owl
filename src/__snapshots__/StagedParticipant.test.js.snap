// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Staged Participant renders the UI as expected 1`] = `
<div
  className="staged-participant"
>
  NAME
</div>
`;
